nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1018,13 +1029,18 @@ protected void inject() throws CommandException {
         // "DEFAULT"
         options.set("DEFAULT", operands);
 
-        // if command has a "terse" option, set it from ProgramOptions
+        // if command has a "terse" or "extraterse" option, set it from ProgramOptions
         if (commandModel.getModelFor("terse") != null){
             options.set("terse", Boolean.toString(programOpts.isTerse()));
         }
+        if (commandModel.getModelFor("extraterse") != null){
```

Comment:
This is for allowing you to specify the option after the command name for CLI commands